qiskit/optimization/algorithms/admm_optimizer.py

```diff
@@ -608,137 +647,76 @@ def update_rho(self, r, s):
 
-    def get_cons_res(self):
+    def get_constraint_residual(self):
         """
         Compute violation of the constraints of the original problem, as:
             * norm 1 of the body-rhs of the constraints A0 x0 - b0
             * -1 * min(body - rhs, 0) for geq constraints
             * max(body - rhs, 0) for leq constraints
+
+        Returns:
+            Violation of the constraints as a float value
         """
 
-        # TODO: think whether a0, b0 should be saved somewhere.. Might move to state?
-        a0, b0 = self.get_a0_b0()
-        cr0 = sum(np.abs(np.dot(a0, self._state.x0) - b0))
+        cr0 = sum(np.abs(np.dot(self._state.a0, self._state.x0) - self._state.b0))
 
-        a1, b1 = self.get_a1_b1()
-        eq1 = np.dot(a1, self._state.x0) - b1
+        eq1 = np.dot(self._state.a1, self._state.x0) - self._state.b1
         cr1 = sum(max(val, 0) for val in eq1)
 
-        a2, a3, b2 = self.get_a2_a3_b2()
-        eq2 = np.dot(a2, self._state.x0) + np.dot(a3, self._state.u) - b2
+        eq2 = np.dot(self._state.a2, self._state.x0) + np.dot(self._state.a3, self._state.u) - self._state.b2
         cr2 = sum(max(val, 0) for val in eq2)
 
-        return cr0+cr1+cr2
+        return cr0 + cr1 + cr2
 
     def get_merit(self, cost_iterate, cr):
         """
         Compute merit value associated with the current iterate
+
+        Args:
+            cost_iterate: cost at the certain iteration
+            cr: value of violation of the constraints
+
+        Returns:
+            merit value as a float
         """
         return cost_iterate + self._mu * cr
 
-    def get_cost_val(self):
+    def get_objective_value(self):
         """
         Computes the value of the objective function.
+
+        Returns:
+            Value of the objective function as a float
         """
+
         # quadr_form = lambda A, x, c: np.dot(x.T, np.dot(A, x)) + np.dot(c.T, x)
         def quadratic_form(matrix, x, c): return np.dot(x.T, np.dot(matrix, x)) + np.dot(c.T, x)
 
-        q0 = self.get_q0()
-        q1 = self.get_q1()
-        c0 = self.get_c0()
-        c1 = self.get_c1()
-
-        obj_val = quadratic_form(q0, self._state.x0, c0)
-        obj_val += quadratic_form(q1, self._state.u, c1)
+        obj_val = quadratic_form(self._state.q0, self._state.x0, self._state.c0)
+        obj_val += quadratic_form(self._state.q1, self._state.u, self._state.c1)
 
         return obj_val
 
-    def get_sol_res(self, it):
+    def get_solution_residuals(self, it):
         """
         Compute primal and dual residual.
 
         Args:
-            it:
+            it: iteration number
+
+        Returns:
+            r, s as primary and dual residuals
         """
         elements = self._state.x0 - self._state.z - self._state.y
         # debug
         # elements = np.asarray([x0[i] - z[i] + y[i] for i in self.range_x0_vars])
         r = pow(sum(e ** 2 for e in elements), 0.5)
```

Comment:
You could just use `np.linalg.norm(elements)` instead (or add a comment that `r` is the norm)